marlon/deepq/simulate.py

```diff
@@ -36,19 +37,17 @@ def run_simulation():
     dql_run = torch.load('deepq.pkl')
     learner = dql_run['learner']
 
-    iteration_count = 1000
+    iteration_number = 1000
```

Comment:
Maybe this should be a constant instead of being returned by this function.

simulate.py
```python
ITERATION_COUNT = 1000
```

web.py
```python
return render_template('index.html', max_sim=simulate.ITERATION_COUNT, list_length=len(SIMULATION)
```

Reply:
I didn't use a iteration count constant in case we want to have that be an option later on to change on the fly. I can definately add it as a constant though

Reply:
Yeah I was actually thinking earlier today it might be better as a parameter to this function and then constant would be defined in `web.py`.

Reply:
Yeah, that way if we wanted it to be specified in the UI, the support would already be in web.py/simulate.py. Ill add the constant in web.py